codeowners-checker.gemspec

```
@@ -14,9 +14,9 @@ Gem::Specification.new do |spec|
   spec.summary       = 'Check consistency of Github CODEOWNERS and git changes.'
   spec.license       = 'MIT'
 
-  spec.files = `git ls-files -z`.split("\x0").reject do |f|
-    f.match(%r{^(test|spec|features)/})
-  end
+  spec.files = Dir.glob("{lib}/**/*", File::FNM_DOTMATCH).reject {|f| File.directory?(f) }
```

Any reason to have `{lib}`?

The second parameter is not interesting since we're not going to cover directories:

```suggestion
  spec.files = Dir.glob("lib/**/*.rb")
```